### PR TITLE
revise omnidir::calibrate API (InputArray to InpurArrayOfArray) 

### PR DESCRIPTION
**Pull request by jiuerbujie**
### This pullrequest changes

revise omnidir::calibrate API (InputArray to InpurArrayOfArray) so that Python bindings will work

### CODE REVIEW

File: modules/ccalib/include/opencv2/ccalib/omnidir.hpp [hunk 1]
```diff
@@ -168,7 +168,7 @@ namespace omnidir
     @param idx Indices of images that pass initialization, which are really used in calibration. So the size of rvecs is the
     same as idx.total().
     */
-    CV_EXPORTS_W double calibrate(InputArray objectPoints, InputArray imagePoints, Size size,
+	CV_EXPORTS_W double calibrate(InputArrayOfArrays objectPoints, InputArrayOfArrays imagePoints, Size size,
```
Comment (sovrasov) [inline modules/ccalib/include/opencv2/ccalib/omnidir.hpp]: Please, replace tab by spaces

**Comment by sovrasov** [comment]
We try to avoid noninformative commits, so it's needed to squash commits into one.

**Comment by jiuerbujie** [comment]
sorry for that, I am not very familiar with these.

**Comment by sovrasov** [comment]
👍

**Pull request merged**